diff -urNp -x '*.orig' uhd-4.1.0.1.org/host/CMakeLists.txt uhd-4.1.0.1/host/CMakeLists.txt
--- uhd-4.1.0.1.org/host/CMakeLists.txt	2021-07-13 17:10:44.000000000 +0200
+++ uhd-4.1.0.1/host/CMakeLists.txt	2021-08-15 16:33:42.068712434 +0200
@@ -458,14 +458,6 @@ endif()
 ########################################################################
 # Create Pkg Config File
 ########################################################################
-foreach(inc ${Boost_INCLUDE_DIRS})
-    list(APPEND UHD_PC_CFLAGS "-I${inc}")
-endforeach(inc)
-
-foreach(lib ${Boost_LIBRARY_DIRS})
-    list(APPEND UHD_PC_LIBS "-L${lib}")
-endforeach(lib)
-
 #use space-separation format for the pc file
 string(REPLACE ";" " " UHD_PC_REQUIRES "${UHD_PC_REQUIRES}")
 string(REPLACE ";" " " UHD_PC_CFLAGS "${UHD_PC_CFLAGS}")
