Build libuhd_test as static to avoid visibility issues
--- uhd-4.1.0.5/host/tests/common/CMakeLists.txt.orig	2022-01-25 21:48:04.385934019 +0100
+++ uhd-4.1.0.5/host/tests/common/CMakeLists.txt	2022-01-25 21:49:45.142054843 +0100
@@ -8,6 +8,6 @@
 # Build uhd_test static lib
 ########################################################################
 include_directories("${UHD_SOURCE_DIR}/lib/include")
-add_library(uhd_test
+add_library(uhd_test STATIC
     ${CMAKE_CURRENT_SOURCE_DIR}/mock_zero_copy.cpp
 )
